docs/explanations/purging_services.md

```diff
@@ -0,0 +1,79 @@
+# Disabling and purging services
+
+All services enabled using the Pro Client can be disabled using the command line:
+
+```bash
+$ sudo pro disable <service-name>
+```
+
+When you disable a service, the Client will remove all access to that specific service's packages and sources in the system. Note that only the access is removed, not the packages (debs, snaps) installed using the service.
+
+For example, the command
+```bash
+$ sudo pro disable esm-apps
+```
+will remove the sources.list entry for esm-apps, and also the authentication information that granted access to the packages. Any deb installed from esm-apps will remain istalled on the system.
+
+On the same line, the command
+```bash
+$ sudo pro disable livepatch
+```
+will disable the livepatch service, but the `canonical-livepatch` snap will still be installed in the system, although inactive.
+
+If there are other services which depend on the service being disabled, those need to be disabled too. A prompt in the CLI will list those and and ask to disable them. The `--assume-yes` flag can be used to automatically accept this prompt.
+
+## What does `purge` mean in this context?
+
+Disabling and *purging* a service will have a similar effect of what is done by [`ppa-purge`](https://launchpad.net/ppa-purge) - it will remove access to the sources from where the service packages can be obtained, and then make best-effort to downgrade those packages back to their Ubuntu versions - the ones published in the Ubuntu Archive.
+
+If a package is specific to the given service (`ubuntu-fips` for the FIPS related services, for example) then it will be removed during the purge process, when possible.
+
+The operation of reverting/uninstalling packages may leave the system in an undesired state, due to packages not meeting dependencies (partial or broken system caches) or to removal of kernels when alternative kernels are not working properly. Therefore, the feature is aimed at users/administrators who understand the impacts.
+
+
+## Using the CLI to disable and purge a service
+
+First of all it is important to note that this is an ***experimental feature***, so it is available but it is advisable to use at one's own discretion.
+
+To purge a service, use the CLI command:
+```bash
+$ sudo pro disable <service-name> --purge
+```
+
+Because of the risks involved, the expectation is that someone is looking at the screen while purging a service. To make it harder to use this feature on scripts, the `--assume-yes` flag is incompatible with the `--purge` flag.
```

Comment:
I'm not sure I like the way this is worded - it makes it sounds like you guys implemented it that way to be awkward/annoying when actually it was a safeguard to prevent accidental damage to users systems, so I'd suggest something more like the following:

We recommend only using `purge` when you are able to monitor the screen. Since the `purge` feature has the potential to do serious accidental damage to a system if used unattended (for example, in a script), the `--assume-yes` flag was made incompatible with the `--purge` flag.

Reply:
accepted